bandwidth_group_accessors.go

```diff
@@ -0,0 +1,89 @@
+package transmissionrpc
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+)
+
+/*
+   		Bandwidth Group Accessors
+        https://github.com/transmission/transmission/blob/main/docs/rpc-spec.md#482-bandwidth-group-accessor-group-get
+*/
+
+var validBandwidthGroupFields []string
+
+func init() {
```

Comment:
Can we reduce this library's `init()` function count to 1 or 0 instead of add to it? This question is for the author and the owner.

This is me being nitpicky, so take what you want from this.  I'm not a huge fan of init functions, and especially when all they do is generate lists that could be created with a bash script called from `go generate`. In other words, generate the code at build time instead of runtime.

Reply:
The idea was to have a single source of truth (the json tags) while keeping it dead simple for the users of the lib (no usage overhead).
I understand your PoV but keeping a single SoT while switching to a static code generation would complexify the usage of the lib more than I want.